pyfr/partitioners/base.py

```diff
@@ -76,6 +59,7 @@ def _combine_mesh_parts(self, mesh):
                 if n > 0:
                     offs[en][i] = off = sum(s.shape[1] for s in spts[en])
                     spts[en].append(mesh[f'spt_{en}_p{i}'])
+                    linf[en].append(mesh[f'spt_{en}_p{i}', 'linear'])
```

Comment:
How about this to make sure we can still partition old meshes?
```
                    # Assume all non-linear elements if the attribute linear
                    # is not present. This ensure backwards compatibility.
                    try:
                        linf[en].append(mesh[f'spt_{en}_p{i}', 'linear'])
                    except KeyError:
                        flags = np.full(mesh[f'spt_{en}_p{i}'].shape[1], False)
                        linf[en].append(flags)
```

Reply:
I'll take this for 1.12, but pending removal in 1.13.